services/ui-src/src/components/cards/EntityCard/QualityMeasuresSection.tsx

```diff
@@ -0,0 +1,65 @@
+import { ReactNode } from "react";
+import { Box, Flex, Text } from "@chakra-ui/react";
+import { SxObject } from "types";
+
+export const QualityMeasuresSection = ({
+  formattedEntityData,
+  printVersion,
+  notAnswered,
+  verbiage,
```

Comment:
since this is the only entity that uses `verbiage` we could import it directly into this file